robottelo/vm.py

```diff
@@ -579,6 +579,25 @@ def configure_rhai_client(self, activation_key, org, rhel_distro):
                 'Unable to register client to Access Insights through '
                 'Satellite')
 
+    def patch_os_release_version(self, distro=None):
```

Comment:
that method seems weird to me as anyway only one distro is supported, but even in that case, next code seems more appropriate:
```
+    def patch_os_release_version(self, distro=DISTRO_RHEL7):
 +        """Patch VM OS release version.
 +
 +        This is needed by yum package manager to generate the right RH
 +        repositories urls.
 +        """
 +        if distro == DISTRO_RHEL7:
```

Reply:
@oshtaier we will add an other distro very soon to install virt-who server on rhel6 :)
and we are using this patch in many places